Extension/src/LanguageServer/client.ts

```diff
@@ -971,6 +971,8 @@ export class DefaultClient implements Client {
         paths.push(localize("installCompiler.string", "Help me install a compiler"));
         paths.push(localize("noConfig.string", "Do not configure a compiler (not recommended)"));
         const index: number = await this.showSelectDefaultCompiler(paths);
+        const action: string = index === -1 ? 'escaped' : index === paths.length - 1 ? 'disable' : index === paths.length - 2 ? 'help' : index === paths.length - 3 ? 'browse' : 'select compiler';
+        telemetry.logLanguageServerEvent('compilerSelection', { action });
```

Comment:
What about escape after browse? Do we want telemetry on that?

Reply:
I think we can address that in a follow up PR.  There are a few telemetry requests in the spec that weren't implemented.  I'd like to get some basic stuff for 1.14.2 without missing our deadline